src/services/chris_integration.tsx

```diff
@@ -354,6 +354,8 @@ class ChrisIntegration {
   }
 
   static async fetchPacFiles(patientID: any): Promise<DcmImage[]> {
+    if (!patientID) return [];
```

Comment:
Would it be entirely incorrect to return all the first page results, when the user submits a blank MRN? I know many other websites that have this functionality implemented on purpose, so that users can see the entire list of results by clicking "search" with empty query parameters